iOS/UI/Reader/Readers/Paged/ReaderPagedViewController.swift

```diff
@@ -574,6 +688,15 @@ extension ReaderPagedViewController: UIContextMenuInteractionDelegate {
                 }
             }
 
+            let isolatePageAction = UIAction(
+                title: NSLocalizedString("AS_ISOLATE_PAGE", comment: ""),
+                image: UIImage(systemName: "rectangle.portrait")
+            ) { _ in
+                Task { @MainActor in
+                    self.isolateCurrentPage(for: pageView)
+                }
+            }
```

Comment:
could we only add this if we're using the double page layout currently? would also be best to not show it if we've already isolated the selected page view (in the case the option does nothing)